File: CartQuestiOS/CartQuestiOS/Models/CompletedRun.swift
import Foundation

struct CompletedRun: Codable, Identifiable {
    var id: String = ""
    var userId: String = ""
    var displayName: String = ""
    var photoUrl: String = ""
    var completedAt: Date = Date()
    var stores: [StoreStop] = []
    var totalDriveTimeMinutes: Int = 0
    var totalCost: Double = 0.0
}

File: CartQuestAndroid/app/src/main/java/com/amoghghadge/cartquestandroid/data/model/CompletedRun.kt
package com.amoghghadge.cartquestandroid.data.model

data class CompletedRun(
    val id: String = "",
    val userId: String = "",
    val displayName: String = "",
    val photoUrl: String = "",
    val completedAt: Long = System.currentTimeMillis(),
    val stores: List<StoreStop> = emptyList(),
    val totalDriveTimeMinutes: Int = 0,
    val totalCost: Double = 0.0
)
